src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java

```diff
@@ -15,17 +22,13 @@
  */
 public class BasePageObject extends MobileBasePageObject {
```

Comment:
I would suggest taking a step back with this architecture:
This class is called BasePageObject. It's risky. 
There is already a BasePageObject class which is considered to be wikia-independent. So it doesn't contain any wikia-related stuff, and that is its purpose.

So the mercury/BasePageObject:
- is a false duplicate, doesn't follow the wikia-independent purpose 
- extends MobileBasePageObject which makes it less "base", when its name suggests being more "base"